x-pack/plugins/security_solution/public/resolver/store/middleware/related_events_fetcher.ts

```diff
@@ -84,29 +80,25 @@ export function RelatedEventsFetcher(
             eventCategory,
             cursor: result.nextEvent,
             nodeID,
-            dataRequestID: newID,
           },
         });
       }
     }
 
     // If the panel view params have changed and the current panel view is either `nodeEventsInCategory` or `eventDetail`, then fetch the related events for that nodeID.
-    if (!isEqual(newParams, oldParams) || selectors.eventsInCategoryResultIsStale(state)) {
+    if (!isEqual(newParams, oldParams)) {
       if (newParams.panelView === 'nodeEventsInCategory') {
         const nodeID = newParams.panelParameters.nodeID;
         api.dispatch({
           type: 'appRequestedNodeEventsInCategory',
           payload: {
             parameters: newParams,
-            dataRequestID: newID,
           },
         });
-        fetchEvents({
+        await fetchEvents({
```

Comment:
Let's add an `await` to the call below to

Reply:
done, thanks